app/system/views/maillayouts/form/form_tabs.blade.php

```diff
@@ -0,0 +1,53 @@
+@php
+$type = $tabs->section;
+$activeTab = $activeTab ? $activeTab : '#'.$type.'tab-1';
+@endphp
+<div class="tab-heading">
+    <ul class="form-nav nav nav-tabs">
+        @php
+            $index = 0;
```

Comment:
use `$loop->iteration` here and avoid using the @php directive, doesn't feel clean to me so we should try avoid using it when we can.